src/iss.js

```diff
@@ -394,6 +396,7 @@ export async function search(
     let request_: searchRequest = {
         q: "",
         type: "service",
+        minimum_must_match: "3>50%",
```

Comment:
I'm not sure if `>` is allowed. Docs suggest `<` or `<-`
https://www.elastic.co/guide/en/elasticsearch/reference/1.4/query-dsl-minimum-should-match.html